File: src/point.cs

public class Point
{
    private double X { get; }
    private double Y { get; }

    public Point(double x, double y)
    {
        this.X = x;
        this.Y = y;
    }

    public double DistanceTo(Point other)
    {
        return Math.Sqrt(
            Math.Pow(this.X - other.X, 2) + 
            Math.Pow(this.Y - other.Y, 2));
    }
}

File: src/point.py
import math

# class syntax
class Point():
    def __init__(self, x: int, y: int) -> None:
        self.x = x
        self.y = y

    # equality
    def __eq__(self, o: object) -> bool:
        if isinstance(o, Point):
            return (self.x == o.x) and (self.y == o.y)
        return False

    def distance_to(self, other: "Point") -> float:
        return math.sqrt(
            (self.x - other.x) ** 2 + 
            (self.y - other.y) ** 2)

